R/spatial_block_cv.R

```diff
@@ -240,6 +248,15 @@ generate_folds_from_blocks <- function(data, centroids, grid_blocks, v, n, radiu
 
   indices <- row_ids_intersecting_fold_blocks(grid_blocks, centroids)
 
+  # could potentially randomly sample to assign indices to folds here
```

Comment:
Did you mean to leave this comment here? I don't quite get what it's trying to say.

Reply:
My idea is that we _could_, rather than erroring here, just assign each observation to a fold at random. If the issue is that an observation is on the edge of two polygons, we'd just pick at random which one it belongs to. Or we could always assign it to the first place it shows up (treating all other folds it's in as a duplicate). The comment is meant as a sort of TODO note, but more as a TO THINK ABOUT, because I don't have a good idea about how to automatically handle "ties" in this situation.